packages/security/src/admin/routes/UpdateSecurityClient.route.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ConduitRoute,
+  ConduitRouteReturnDefinition,
+  PlatformTypesEnum,
+} from '@conduitplatform/commons';
+import {
+  ConduitError,
+  ConduitRouteActions,
+  ConduitRouteParameters,
+  ConduitString,
+  RouteOptionType,
+} from '@conduitplatform/grpc-sdk';
+import { Client } from '../../models';
+import { isNil } from 'lodash';
+
+export function getUpdateSecurityClientRoute() {
+  return new ConduitRoute(
+    {
+      path: '/security/client/:id',
+      urlParams: {
+        id: ConduitString.Required,
+      },
+      action: ConduitRouteActions.UPDATE,
+      bodyParams: {
+        platform: ConduitString.Optional,
```

Comment:
platform is not an acceptable argument. As we've discussed the user cannot change the platform